EasyPost/Services/EndShipperService.cs

```diff
@@ -20,25 +27,12 @@ internal EndShipperService(EasyPostClient client)
         #region CRUD Operations
 
         /// <summary>
-        ///     Create an EndShipper.
+        ///     Create an <see cref="EndShipper"/>.
+        ///     <a href="https://www.easypost.com/docs/api#create-an-endshipper">Related API documentation</a>.
         /// </summary>
-        /// <param name="parameters">
```

Comment:
question: why remove all the valid options, seems like a regression?

Reply:
Link to the web docs provided, so we don't need to keep this docstring updated, users can visit the web docs for the up-to-date source of truth.